examples/frustum-culling/main.js

```diff
@@ -487,7 +488,7 @@ bs.getOrCreateStateSet().setTextureAttributeAndModes( 0, new osg.Texture(), osg.
                 'uniform mat4 uModelViewMatrix;',
                 'uniform mat4 uProjectionMatrix;',
                 'void main(void) {',
-                '  gl_Position = uProjectionMatrix * uModelViewMatrix * vec4(Vertex,1.0);',
+                '  gl_Position = uProjectionMatrix * (uModelViewMatrix * vec4(Vertex,1.0));',
```

Comment:
Those parenthesis tranforms a  mat4*mat4 into a  mat4*vec4.
It's needed everywhere we does this to prevent further copy and past of the unoptimized code (mat4*mat4).